components/VTheme/Profile.js

```diff
@@ -1,31 +1,56 @@
 import styled from 'styled-components'
 
 export const ProfileBanner = styled.header`
+margin: 5rem 0 0rem 0;
 display: grid;
 grid-template-columns: 100%;
 align-self: center;
 justify-self: center;
-
+margin-bottom: 2rem;
 @media screen and (min-width: 767px) {
-  grid-template-columns: 25% 75%;
+  grid-template-columns: 15rem 1fr;
+  gap: 5rem;
+ 
 }
 
 img {
-  width: 16rem;
-  max-height: 12rem;
+  width: 20rem;
+  height: 20rem;
   object-fit: cover;
   align-self: center;
   justify-self: center;
+  padding: 2.5rem;
+  @media screen and (max-width: 767px) {
+    width: 15rem;
+  height: 15rem;
+ padding: 1rem;
+
+ 
+}
 }
 `
 export const ProfileBannerTitle = styled.div`
 
+align-self: center;
+text-align: center;
+
+
 @media screen and (min-width: 767px) {
-  padding-left: 3rem;
+
   text-align: left;
 }
 
 `
+
+export const ProfileBannerImageContainer = styled.div`
```

Comment:
suggest rename to VBannerImage and make it a styled.img. Then you don't need to wrap the image in the styling but can just have <VBannerImage src=etc   cleaner code. 
This makes it clearer that we mean the top left image on the banner.

Reply:
might style img inside the VBanner instead

Reply:
Sussed